app/models/hold.rb

```diff
@@ -53,10 +53,10 @@ def created_by
   # This may later list just the info for the file flagged 'primary', once 
   # we implement that feature.
   def dates_thesis_files_received
-    if self.thesis.files.present?
+    if self.thesis.present? && self.thesis.files.present?
       self.thesis.files.map do |file| 
         "#{file.created_at.strftime('%Y-%m-%d')} (#{file.blob.filename})"
-      end.join("\n")
+      end.join("; ")
```

Comment:
Yeah, I completely agree. I changed the delimiter to a semicolon because the newline character doesn't parsed within the `value` attribute of the form element. The good news is that in the future, this field will become 'Date thesis file received' and list only the primary file. (This was the original intention, but we don't yet have the means to designate a primary file.)